0 size Tensor√ü
paddle.gather(x=Tensor([3, 4],"float64"), index=Tensor([0],"int32"), axis=0, )paddle.gather(x=Tensor([3, 4],"float64"), index=Tensor([0],"int32"), axis=0, )

aistudio@jupyter-3165291-8843119:~/PaddleAPITest$ python engine.py --paddle_only=True --api_config='paddle.gather(x=Tensor([3, 4],"float64"), index=Tensor([0],"int32"), axis=0, )'
test begin: paddle.gather(x=Tensor([3, 4],"float64"), index=Tensor([0],"int32"), axis=0, )
W0409 08:45:17.809623 165008 gpu_resources.cc:119] Please NOTE: device: 0, GPU Compute Capability: 7.0, Driver API Version: 12.0, Runtime API Version: 11.8
W0409 08:45:17.810878 165008 gpu_resources.cc:164] device: 0, cuDNN Version: 8.9.
[cuda error] paddle.gather(x=Tensor([3, 4],"float64"), index=Tensor([0],"int32"), axis=0, ) 
 (External) CUDA error(9), invalid configuration argument. 
  [Hint: 'cudaErrorInvalidConfiguration'. This indicates that a kernel launch is requesting resources that can never be satisfied by the current device. Requestingmore shared memory per block than the device supports will trigger this error, as will requesting too many threads or blocks.See cudaDeviceProp for more device limitations.] (at ../paddle/fluid/pybind/eager_functions.cc:1388)